geoapi/tasks/streetview.py

```diff
@@ -108,7 +115,7 @@ def _from_tapis(user: User, task_uuid: UUID, systemId: str, path: str, organizat
         try:
             img_name = os.path.join(str(base_filepath), Path(item.path).name)
             img_list.append(img_name)
-            client.getRawFileToPath(systemId, item.path, img_name)
+            client.get_file_to_path(systemId, item.path, img_name)
```

Comment:
this is the only actual change to the file if you ignore linting and the addition of a docstr and a change in logging.